src/StatusBarMenu.ts

```diff
@@ -134,7 +130,7 @@ export class StatusBarMenu {
         this._isLoading = true;
         this.refreshStatusItem();
         if (this._currentKubeconfigContext != null) {
-            const kubectlClient = await this._binariesUtility.tryGetKubectlAsync();
+            var kubectlClient;
```

Comment:
💡We can do this refactor in more smaller junk to be sure we are not adding any side effects, Have you tested this by deleting any previous b2k binary because this seems little interesting, for example, the `var` is introduced here but instantly in line below, an empty variable is sent to another function.